sys/sys/user.h

```diff
@@ -576,7 +576,8 @@ struct kinfo_vmentry {
 	} kve_type_spec;
 	uint64_t kve_vn_rdev;			/* Device id if device. */
 	uint64_t kve_reservation;		/* Map reservation */
-	int	 _kve_ispare[6];		/* Space for more stuff. */
+	int	 kve_max_protection;		/* Max protection bitmask. */
```

Comment:
I wonder if we should upstream kve_max_protection and swap them around in CheriBSD before the next release?

Reply:
https://reviews.freebsd.org/D46293